test/dsl/dsl_model_construction.jl

```diff
@@ -10,8 +10,9 @@ t = default_t()
 using StableRNGs
 rng = StableRNG(12345)
 
-# Fetch test networks.
+# Fetch test networks and functions.
 include("../test_networks.jl")
+include("../test_functions.jl")
```

Comment:
In the end I could not pu these "test helper functions" in the runtests.jl file. I created a separate file, which is fetched whenever it is needed (like here).